emukit/examples/fabolas/example.py

```diff
@@ -0,0 +1,22 @@
+from hpolib.benchmarks.surrogates.svm import SurrogateSVM
```

Comment:
We don't have a dependecy on hpolib, and we shouldn't, so what we do in such cases is wrap this import in try/except and print a friendly message, something like "hpolib is required for this example, run `pip install hpolib`" or something of that sort